pegasus/sites.v3/code.org/public/css/hoc-banner.scss

```diff
@@ -0,0 +1,116 @@
+@import 'color.scss', 'font.scss', 'imports/_breakpoints.scss';
```

Comment:
Is this the same changes as in https://github.com/code-dot-org/code-dot-org/pull/48349/files?

Reply:
Ah, I see, one lives in the hourofcode.com directory and this lives in code.org!

Reply:
They're very similar changes, but a little different due to how it's implemented.

Reply:
I see. I wonder, since we'll be reviewing these PRs in the future, if it'd be worth it for us to meet with you to figure out how this land works? I see slight differences between the code.org and hourofcode.com content here –– but besides the banners themselves being different images, what other styling is generally similar and different?

Reply:
I'm definitely up for chatting 👍 

The gist of it though is that the code.org homepage banner uses Ruby-based logic to create the banner (Brendan helped set this up) since the banner is more variable based on promo campaigns and will be switched out halfway through HoC season w/ a different promo banner that'll very likely require updated styling. 

The hoc.com banners are more evergreen since we'll likely just update once a year for `soon-hoc` and then replace the banner each year based on the new concept/theme.

Reply:
Ah, that's very helpful –– thank you for the context!

Reply:
Yw! 😄